static/app/types/core.tsx

```diff
@@ -67,12 +67,11 @@ export type Choice = [
 export type Choices = Choice[];
 
 /**
- * This legacy type used plurals which will cause compatibility issues when categories
- * become more complex, e.g. processed transactions, session replays. Instead, access these values
- * with `DATA_CATEGORY_INFO[category].plural`, where category is the `DataCategoryExact` enum value.
+ * These are very similar to the plural types of DATA_CATEGORY_INFO.
+ * DATA_CATEGORY_INFO and DataCategoryExact have additional categories
+ * that are used in stats but not other places like billing.
  */
 export enum DataCategory {
-  DEFAULT = 'default',
```

Comment:
What is the default for?

Reply:
it seems unused, but we can keep it for now

Reply:
We have a DataCategory called DEFAULT. It gets used for error events that don't have stack traces like from `capture_message`. Default events consume error quota.

Reply:
Perhaps we should investigate further and remove `DataCategory.DEFAULT` in a separate PR if necessary.